src/dialog-user/components/dialog-user/dialogField.html

```diff
@@ -48,19 +48,38 @@
               id="{{ vm.dialogField.name }}">
         <div ng-if="vm.dialogField.fieldValidation===false">{{vm.dialogField.errorMessage}}</div>
       </div>
+
       <span ng-switch-when="DialogFieldDropDownList">
-        <ui-select ng-model="vm.dialogField.selected"
-                   ng-blur="vm.validateField()"
-                   on-select="vm.changesHappened($item)"
-                   ng-disabled="vm.dialogField.read_only || vm.inputDisabled"
-                   theme="bootstrap"
-                   input-id="{{ vm.dialogField.name }}">
-          <ui-select-match placeholder="{{vm.dialogField.description}}">{{$select.selected[1] }}</ui-select-match>
-          <ui-select-choices
-            repeat="fieldValue[0] as fieldValue in vm.dialogField.values | filter: $select.search">
-            <span>{{fieldValue[1]}}</span>
-          </ui-select-choices>
-        </ui-select>
+        <!-- Dropdown field where a single value is expected -->
+        <select pf-select
+                ng-if="!vm.dialogField.options.force_multi_value"
+                ng-model="vm.dialogField.default_value"
+                ng-change="vm.changesHappened(item)"
```

Comment:
Pretty sure `item` is undefined here.. `$item` used to be provided by `ui-select`, but looks like `changesHappened` does `(value ? value[selectedValue] : this.dialogField.default_value)`, so maybe you can drop it and simplify the function a bit?